kommunicate/src/main/java/com/applozic/mobicomkit/api/conversation/MobiComMessageService.java

```diff
@@ -186,6 +186,10 @@ public Contact addMTMessage(Message message, int index) {
             message.setHidden(true);
         }
 
+        if(message.isFeedbackMessage()) {
+            message.setHidden(true);
```

Comment:
Change it to one line, message.setHidden( message.isFeedbackMessage() );

Reply:
Okay, will do